main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
The requirements say to loop 10 times, this is 8. If you wanted to use a while loop, you could set `loop = 0` on line 3 and on line 4, change the while loop to `while (loop < 10)`.